docs/ref/models/fields.txt

```diff
@@ -1810,10 +1810,16 @@ relation. These attribute are present on all fields; however, they will only
 have meaningful values if the field is a relation type
 (:attr:`Field.is_relation=True <Field.is_relation>`).
 
+.. attribute:: Field.many_to_one
```

Comment:
I figured but for documentation I think it's quite confusing to start with the exceptions "reverse fields" and something from contrib.

Here we start with the forward field for FK, followed by its reverse, that's IMO more logical, what do you think?